packages/modeling/src/operations/transforms/rotate.js

```diff
@@ -15,11 +15,14 @@ const { geom2, geom3, path2 } = require('../../geometry')
  */
 const rotate = (angles, ...objects) => {
   if (!Array.isArray(angles)) throw new Error('angles must be an array')
-  if (angles.length !== 3) throw new Error('angles must contain X, Y and Z values')
 
   objects = flatten(objects)
   if (objects.length === 0) throw new Error('wrong number of arguments')
 
+  // adjust the angles if necessary
+  angles = angles.slice() // don't modify the original
```

Comment:
nice !

Reply:
hmm looking at the examples: can we perhaps add a toArray(angles) , so that even single values can be pushed into angle arrays ?
aka rotate(Math.PI/2, geometry) would also work ?
(I am less 100% sure about this api wise)

Reply:
Hmm... the aliases have single values, e.g. rotateX(Math.PI/2, geometry)

So, I'm not so sure about this change. It would be a rare case to rotate objects the same about X,Y,Z.

Looking at all of the transforms, the only one that might make sense is scale() by a single factor.

Reply:
ok, so perhaps it makes sense to go about it the opposite way: why allow 1d or 2d arrays when when need 3d arrays all the time ? ie, perhaps we can save a bit of complexity and have some additional clarity by ONLY allowing 3 component arrays

Reply:
I think these changes were requested because transforming 2D objects should only require 2 values. That makes sense.

My changes don’t care... any size array is fine. Is that his confusing to users?

Reply:
very good point ! hmm let me think about it until tomorrow please.

Reply:
meh you are right, your changes are good !